docs/site/reference.md

```diff
@@ -119,7 +119,7 @@ patient data and used during query retrieval. The following blocking key types a
 
 `IDENTIFIER` (ID: **10**)
```

Comment:
This Identifier refers to `IDENTIFIER:<type>` listed above as a feature, right? So we can only block on Identifiers if they have a type associated with them?

Reply:
Got it, thank you!